assignments/number-function-practice/code.js

```diff
@@ -0,0 +1,66 @@
+const sum = (a, b) => {
+  return a + b
+}
+const difference = (a, b) => {
+  return a - b
+}
+const product = (a, b) => {
+  return a * b
+}
+const ratio = (a, b) => {
+  return a / b
+}
+const remainder = (a, b) => {
+  return a % b
+}
+const average = (a, b) => {
+  return (a + b) / 2
+}
+
+const twoDistance = (x1, x2, y1, y2) => {
+  return Math.sqrt(Math.abs(x1 - x2) * Math.abs(x1 - x2) + Math.abs(y1 - y2) * Math.abs(y1 - y2))
```

Comment:
This is correct but you could make it more concise if you used the `**` exponentiation operator. Also you don't actually need the `Math.abs` since you're squaring the difference which will ensure that it's positive.

```suggestion
  return Math.sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2)
```

Reply:
i mainly didn't use the ** operator since for some reason it hasn't worked for me at some times before (100% human error). i also just threw in the Math.abs just to make sure.